gen/definitions/interface_group.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: Interface Group
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/object/interfacegroups
+data_source_name_query: true
```

Comment:
If we add this:

```yaml
test_tags: [TF_VAR_device_id]
```

.. it will check if the env var is configured and otherwise log a message. (see `device_vni_interface.yaml` for example)

Reply:
Sure, I'll add this